clips/popup-banner/clip/clip.html

```diff
@@ -1,66 +1,134 @@
 <div class="container">
   <div class="info-wrapper">
-    <div class="info">
-      <div class="details-wrapper">
-        <div class="abstract-lines2-wrapper">
-          <svg
-            width="119"
-            height="8"
-            viewBox="0 0 119 8"
-            fill="none"
-            xmlns="http://www.w3.org/2000/svg"
+    <div class="info-cta">
+      <div class="info">
+        <div class="details-wrapper">
+          <img class="logo" src="{{initParams.logo.default}}" alt="logo" />
```

Comment:
can you add support for additional sizes here based on these examples ? https://css-tricks.com/a-guide-to-the-responsive-images-syntax-in-html/